packages/link-modules/src/utils/index.ts

```diff
@@ -0,0 +1,12 @@
+import { MODULE_RESOURCE_TYPE } from "@medusajs/types"
+
+export * from "./compose-link-name"
+export * from "./generate-entity"
+
+export function shouldForceTransaction(target: any): boolean {
+  return target.moduleDeclaration?.resources === MODULE_RESOURCE_TYPE.ISOLATED
+}
+
+export function doNotForceTransaction(): boolean {
```

Comment:
comment: Every time I see this function used, it feels off. Is this just me? 😅 

Does this have anything to do with forcing? Wouldn't it be more appropriate to have: `useTransactionManagerIfPresent` and return true? 🤔

Reply:
It feels a bit off to me as well, we can revisit these function names and replace that everywhere in another PR. This is being used across all modules and it is indeed confusing.